.meta.toml

```diff
@@ -33,12 +30,9 @@ testenv-deps = [
     ]
 
 [coverage]
-fail-under = 0
+fail-under = 91.66
 
 [github-actions]
-additional-config = [
-    "- [\"2.7\",   \"py27\"]",
-    ]
 additional-install = [
     "pip install -U \"tox < 4\"",
```

Comment:
This should also be removed. It was only necessary because tox 4.x no longer supports Python 2.